lib/collections/unordered-set.d.ts

```diff
@@ -1,12 +1,12 @@
 import { Bytes } from "../utils";
 import { Vector } from "./vector";
 export declare class UnorderedSet {
-    readonly length: number;
     readonly prefix: Bytes;
     readonly elementIndexPrefix: Bytes;
     readonly elements: Vector;
     constructor(prefix: Bytes);
-    len(): number;
+    get length(): number;
+    set length(_l: number);
```

Comment:
There shouldn't be a set method here. That would lead to bugs

Reply:
This is required for our `deserialize` to work, see this comment: https://github.com/near/near-sdk-js/pull/180/files#diff-f67afbdc9e6ba66abb7302662cfab6bd61080e3a7e7247973f3c87bf8ab36d20R33

Reply:
interesting, okay, my assumption was that this could be at least private. All I'm pointing out is that this might be a footgun since this is part of the exposed API to developers

Reply:
Good point! I try to add private to the setter, that still builds and will technically make it not exposed API